src/components/dashboard/utils/sendFromQRCode.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import { NETWORK_ID } from '../../../lib/constants/network'
+
+export type CodeType = { networkId: string, address: string, amount: number }
+
+/**
+ * Returns a function that will decide where to navigate depending on the screen specified
+ * @param {screen} screen
+ * @returns {Function}
+ */
+export const sendFromQRCode = (screen: string): Function =>
+  /**
+   * Curried function that decides where to navigate based on the content of the code and the screen provided
+   * @param {object|null} code
+   * @returns {*[]}
+   */
+  (code: CodeType | null): any => {
+    if (code === null) {
+      throw new Error('Invalid QR Code.')
+    }
+
+    const { networkId, address, amount } = code
+
+    if (networkId !== NETWORK_ID.FUSE) {
```

Comment:
the check should be if networkId!==wallet.networkId (not sure whats the right method in web3 to extract the id)

Reply:
@sirpy you can check in https://github.com/GoodDollar/GoodDAPP/pull/106/commits/b1b3124e106cf70273c91a6c59b7e6875836395d for the specific change.

Other commits are to update implementation and tests.